idea/testData/inspections/forEachParameterNotUsed/test.kt

```diff
@@ -0,0 +1,9 @@
+fun test() {
+    val items = listOf<Any>()
+    items.forEach { }
+    items.forEach { item -> }
+    items.forEach { doSomething(it) }
+    items.forEach { item -> doSomething(item) }
+}
```

Comment:
I'd add some tests with nested `forEach`, to make sure we handle correctly situations with multiple `it`.

Reply:
Validated nested calls. No logic needed to change to support this, it looks like the scope was already being handled properly.